web-ui/src/common/snackbar_notification/snackbar_notification.scss

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017 ThoughtWorks, Inc.
+ *
+ * Pixelated is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Pixelated is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Pixelated. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//TODO: Refer to Issue - https://github.com/callemall/material-ui/issues/3860
+#snackbar > div {
+  line-height: 20px !important;
+  padding: 24px !important;
```

Comment:
We spent a lot of time trying to get our styles to override the default material styles but, because of a multi-line problem with the existing snackbar, we've had to put this `!important` here to override their default mobile styles.  We've referred to an issue on their repo which should be available in their next release hopefully that supports multi-line messages.